### PR TITLE
Add logo for IUC

### PR DESCRIPTION
**Pull request by bebatut**
The possible logo for IUC

iuc_logo

### CODE REVIEW

**Comment by martenson** [comment]
I like it very much. 💯

**Comment by peterjc** [comment]
Nice space theme 👍

**Comment by gregvonkuster** [comment]
👍

**Comment by shiltemann** [comment]
looks great @bebatut! :+1:

**Comment by bgruening** [comment]
:tada: :+1:

**Comment by mvdbeek** [comment]
🎉

File: iuc_logo.svg [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   width="141.08534mm"
+   height="87.072601mm"
+   viewBox="0 0 499.9087 308.52496"
+   id="svg2"
+   version="1.1"
+   inkscape:version="0.91 r13725"
+   sodipodi:docname="iuc_draft.svg"
+   inkscape:export-filename="/Users/bebatut/Documents/com/images/logos/iuc_draft.png"
```
Comment (yhoogstrate) [inline iuc_logo.svg]: I know inkscape did this ;)

**Pull request merged**